.eslintrc.js

```diff
@@ -47,7 +51,36 @@ module.exports = {
       plugins: ['node'],
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
         // add your custom rules and overrides for node files here
+        "@typescript-eslint/explicit-function-return-type": ["off"],
+        "@typescript-eslint/no-var-requires": ["off"],
+        "no-dupe-else-if": ["off"],
+        "no-import-assign": ["off"],
+        "no-setter-return": ["off"],
+        "@typescript-eslint/camelcase": ["off"]
       })
-    }
+    },
+    {
+      files: ["*.ts"],
+      rules: {
+        "@typescript-eslint/explicit-function-return-type": [
+          "error",
+          {
+            allowExpressions: true,
+            allowHigherOrderFunctions: true
+          }
+        ],
+        "no-dupe-else-if": ["off"],
+        "no-import-assign": ["off"],
+        "no-setter-return": ["off"],
```

Comment:
Seems like these are changed in all scenarios, can you remove them from overrides and hoist to the top level `rules`?

Reply:
Ah, yes. 👍 I didn't realize it ended up hitting them all. I need to figure out _why_ these are an issue as well. 🤔